src/main/resources/templates/schedule/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+
+<head>
+  <title>My Class Schedule</title>
+  <div th:replace="fragments/bootstrap_head.html"></div>
+</head>
+
+<body>
+  <div class="container">
+    <div th:replace="fragments/bootstrap_nav_header.html"></div>
+    <h3>My Class Schedules</h3>
+    <select class="custom-select">
+      <option th:each="s:${myschedules}" th:value="${s.scheduleid}" th:text="${s.schedulename}"></option>
+    </select>
+    <!-- th:field="*{scheduleselect}" -->
+
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Quarter</th>
+          <th>Course Name</th>
+          <th>Enroll Code</th>
+          <th>SCHEDULEid</th>
```

Comment:
This is probably unnecessary to the end user - we should look to remove this in a future issue.